mypy_django_plugin/transformers/managers.py

```diff
@@ -154,9 +155,11 @@ def resolve_manager_method(ctx: AttributeContext) -> MypyType:
     A 'get_attribute_hook' that is intended to be invoked whenever the TypeChecker encounters
     an attribute on a class that has 'django.db.models.BaseManager' as a base.
     """
-    # Skip (method) type that is currently something other than Any
+    # Skip (method) type that is currently something other than Any of type `implementation_artifact`
     if not isinstance(ctx.default_attr_type, AnyType):
         return ctx.default_attr_type
+    elif ctx.default_attr_type.type_of_any != TypeOfAny.implementation_artifact:
+        return ctx.default_attr_type
```

Comment:
I'm seeing some issues due to this line, why did you add this?

Reply:
Idea was to guarantee only processing Anys from the dynamic manager class creation (read: methods). So we won't accidentally do something with stuff that isn't manager/queryset methods

Also: https://github.com/typeddjango/django-stubs/pull/1025#discussion_r910734539

Reply:
Ah, then I guess I'm doing something in one of my branches where I'm not correctly setting the Any type :)